CHANGELOG.md

```diff
@@ -624,7 +624,7 @@ A brief description of the categories of changes:
   default, which will cause `gazelle` to change third-party dependency labels
   from `@pip_foo//:pkg` to `@pip//foo` by default.
 
-* The `compile_pip_requirements` now defaults to `pyproject.toml` if the `src`
+* The `compile_pip_requirements` now defaults to `pyproject.toml` if the `srcs`
```

Comment:
You should have a new changelog entry instead of modifying old entries. What you are doing is a breaking change here.

Reply:
Ugh, yep. When this was originally created, `src` hadn't been published 😭

Reply:
Should we deprecate `src` in favor of `srcs`?